e2e/tests/visual-a11y/a11y.visual.spec.js

```diff
@@ -0,0 +1,34 @@
+/*****************************************************************************
+ * Open MCT, Copyright (c) 2014-2023, United States Government
+ * as represented by the Administrator of the National Aeronautics and Space
+ * Administration. All rights reserved.
+ *
+ * Open MCT is licensed under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * http://www.apache.org/licenses/LICENSE-2.0.
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ *
+ * Open MCT includes source code licensed under additional open source
+ * licenses. See the Open Source Licenses file (LICENSES.md) included with
+ * this source code distribution or the Licensing information page available
+ * at runtime from the About dialog for additional information.
+ *****************************************************************************/
+
+const { expect, test, generateAccessibilityReport } = require('../../avpFixtures');
+const VISUAL_URL = require('../../constants').VISUAL_URL;
+
+test.describe('a11y - Default @a11y', () => {
+  test.beforeEach(async ({ page }) => {
+    await page.goto(VISUAL_URL, { waitUntil: 'domcontentloaded' });
+  });
+  test('main view @a11y', async ({ page }, testInfo) => {
+    const accessibilityScanResults = await generateAccessibilityReport(page, testInfo.title);
+    expect(accessibilityScanResults.violations).toEqual([]);
```

Comment:
Could try doing something like this
```suggestion
    expect(accessibilityScanResults.violations).toEqual(expect.arrayContaining([]));
```

Reply:
done